### DISCUSSION TITLE
organizzazione fisica - 8

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#87] [Esercizi]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/3dcdf88f-7114-4f2d-8dfe-c10382edd5bb)

**Comment by atyion** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/101365532/ba908ccd-06a7-4d89-a3d9-6c3ccee792be)

**Comment by 7Andrei** [comment]
Aggiungo risultato del secondo punto che usando le formule al contrario vengono massimo 4.128.768 Record
